bootstrap.sh

```diff
@@ -1 +1,38 @@
 #!/bin/bash
+
+#!/bin/bash
```

Comment:
There is a redundant shebang line here. The script should only have one shebang line at the top. You can remove this line.